src/services/config/loader.py

```
@@ -67,8 +79,49 @@ def load_config_with_main(config_file: str, project_root: Path | None = None) ->
     main_config_path = config_dir / "main.yaml"
     if main_config_path.exists():
         try:
-            with open(main_config_path, encoding="utf-8") as f:
-                main_config = yaml.safe_load(f) or {}
+            main_config = _load_yaml_file(main_config_path)
+        except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)